static/app/components/eventOrGroupExtraDetails.tsx

```diff
@@ -73,10 +74,13 @@ function EventOrGroupExtraDetails({
   } = data as Group;
 
   const issuesPath = `/organizations/${organization.slug}/issues/`;
+  const {getReplayCountForIssue} = useReplayCountForIssues();
 
   const showReplayCount =
     organization.features.includes('session-replay') &&
-    projectCanLinkToReplay(organization, project);
+    projectCanLinkToReplay(organization, project) &&
+    data.issueCategory &&
+    getReplayCountForIssue(data.id, data.issueCategory);
```

Comment:
can we add a ` > 0` here - not that it'll make a big different but at least it'll keep `showReplayCount` a boolean

Reply:
Good point about keeping it a boolean - that function also can return `undefined`, so i think just converting it to a boolean would achieve the same result without having to do a separate existence check.